sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -71,16 +71,25 @@ private TestProxy(string proxyPath)
                 EnvironmentVariables =
                 {
                     ["ASPNETCORE_URLS"] = $"http://{IpAddress}:0;https://{IpAddress}:0",
-                    ["Logging__LogLevel__Microsoft"] = "Information"
+                    ["Logging__LogLevel__Microsoft.Hosting.Lifetime"] = "Information",
+                    ["ASPNETCORE_Kestrel__Certificates__Default__Path"] = Path.Combine(
```

Comment:
Will this handle the case that the cert isn't here and/or prompt you to install it?

Reply:
With this change you won't have to install the cert anymore as we just specify the location of the cert that is in the repo.